packages/page-objects/src/components/editor/TextEditor.ts

```diff
@@ -626,6 +626,32 @@ export class TextEditor extends Editor {
 		return new Breakpoint(breakpoints[0], lineElement);
 	}
 
+	/**
+	 * Get all breakpoints.
+	 * @returns List of Breakpoints.
+	 */
+	async getBreakpoints(): Promise<Breakpoint[]> {
+		const breakpoints: Breakpoint[] = [];
+
+		const breakpointLocators = Breakpoint.locators.TextEditor.breakpoint;
+		const breakpointContainer = TextEditor.versionInfo.version >= '1.80.0' ? await this.findElement(By.className('glyph-margin-widgets')) : this;
+		const breakpointsSelectors = await breakpointContainer.findElements(breakpointLocators.generalSelector);
+
+		for (let i = 0; i < breakpointsSelectors.length; i++) {
+			let lineElement: WebElement;
+			if (TextEditor.versionInfo.version >= '1.80.0') {
```

Comment:
same case as above? maybe not whole condition, but consider also this piece of code please 🙂